### PR TITLE
Old Apple Website features

### PR DESCRIPTION
**Pull request by warblo001**
In this project we were tasked to create a clone from the Old Apple Website, using frontend technologies like gradients and backgrounds.
The project has the following features:
- Navigation Bar with gradients
- Backgrounds features on all the project
- Flexbox

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+- Backgrounds features on all the project
+- Flexbox
+
+
+## Built With
+
+- HTML
+- CSS/SCSS
+
+
```
Comment (javitocor) [inline README.md]: [OPTIONAL]
You can add SASS here, remember to add all the techs you use in the readme to highlight it

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+[Source Page Link](https://web.archive.org/web/20140301004610/http://www.apple.com/)
+
+## Authors
+
+😎 **Helcio André**
+
+- GitHub: [@helciodev](https://github.com/helciodev)
+- Twitter: [@helcio_bruno](https://twitter.com/helcio_bruno)
+
```
Comment (javitocor) [inline README.md]: [OPTIONAL]

Add your LinkedIn link, recruiters will appreciate it

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Great job with this project, team! 👍 
Your navbar is impressive, an amazing job with it looks like the original 👏 
Good and descriptive commits and a good readme file 💯 
Good use of backgrounds and gradients 🥇 
Your code is clean 🔝  and well structured 👍 
Linters are green 🟢 

However, we must do some changes in order to get the project approved.
Check my comments below 👇  apply the changes and request for a new code review ⏩ 

Good luck with it 😃 

## Changes

- As I said, the navbar is amazing, but the positioning it is not correct, check the difference with the original and kindly modify it. You can use background-size and position properties.
Original:
Capture

Your project:
Capture1

- Good use of flex, but one of the mandatory requirements is that you have to use float as well.   
       Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

**Review by arslanbisharat — APPROVED** [review]
## Approved
Thank you for making those changes requested by previous TSE. 
This project is approved.
Regards,
Arslan Bisharat

**Pull request merged**